### ISSUE TITLE
Балкани - Structural block "Albania Tirana" (country Albania) is located AFTER the structural block "Road: Albania – Montenegro" (Change the country from "Albania" to "Montenegro")

### ISSUE THREAD

**Issue by DmitriiNasonov** [scholokov/long-travel-2#3893]
**Steps of reproducing**
1. Open https://test.long-travel.live/20190720_0811_balcans
2. Scroll to structural block "Albania Tirana"
3. Pay attention to location of the structural block "Albania Tirana"

**Actual Result**
Structural block "Albania Tirana" (country Albania) is located AFTER the structural block "Road: Albania – Montenegro" (Change the country from "Albania" to "Montenegro")

**Expected Result**
Structural block "Albania Tirana" (country Albania) is located BEFORE the structural block "Road: Albania - Montenegro" (Change the country from "Albania" to "Montenegro")

**Enviroments**
Windows 10, Chrome 114.0.5735.199 (64 bit)
Android 8.1.0, Chrome 114.0.5735.131
![Балкани – Long-Travel - Google Chrome 28 06 2023 14_10_04](https://github.com/scholokov/long-travel-2/assets/95171672/8e87141b-624e-4cd9-bb4e-9361ba7703e5)
![photo_2023-06-28_14-10-41](https://github.com/scholokov/long-travel-2/assets/95171672/57e8b160-349e-4b87-9a1b-4842f1cef523)

**Comment by DmitriiNasonov** [comment]
#233

**Comment by scholokov** [comment]
у даному випадку це нормально. Тірана була по дорозі з Албанії у Монтенегро
![image](https://github.com/scholokov/long-travel-2/assets/22824947/b15688ae-367c-44a6-9bd4-21fe5caa5f9d)

**Comment by DmitriiNasonov** [comment]
Не согласен, структурный блок "Road" предполагает перемещение из одного города в другой город, в данном случае из Vlora (Albania) в Dobra Voda (Montenegro). Но, если путешественники покидают Албанию что-бы путешествовать в Montenegro, то вместо города Vlora лучше будет указать последний посещённый город Албании, чем и является Tirana
![4  Подорож · scholokov_long-travel-2 Wiki - Google Chrome 03 07 2023 13_56_08](https://github.com/scholokov/long-travel-2/assets/95171672/97ba0b89-a0f4-4375-ab89-88900307d7c0)
